Fly.md

```diff
@@ -0,0 +1,10 @@
+### FLY
```

Comment:
move to `projects/javascript/Fly.md`

Reply:
Hey Professor,

I plan to use Python for this Project.